sources/packages/backend/apps/queue-consumers/src/processors/schedulers/base-scheduler.ts

```diff
@@ -47,9 +49,23 @@ export abstract class BaseScheduler<T> implements OnApplicationBootstrap {
    * any old cron job delete it and add the new job to the queue.
    */
   async onApplicationBootstrap(): Promise<void> {
+    const isFulltimeAllowed = this.configService.isFulltimeAllowed;
     await this.deleteOldRepeatableJobs();
     // Add the cron to the queue.
     await this.schedulerQueue.add(await this.payload());
+    // Allow Fulltime Schedulers only if isFulltimeAllowed is true
+    if (
+      !isFulltimeAllowed &&
+      [
+        QueueNames.FullTimeMSFAAIntegration,
```

Comment:
IER and ECE schedulers are also pertinent to full time. I would say we can add them to the list instead of them running dummy schedules not picking any data.